configs/sst_cs_apps-apps-unsorted.yaml

```diff
@@ -10,8 +10,6 @@ data:
   - time
   - pcre2
   - pcre2-devel
-  - pcre
```

Comment:
This is included in CentOS Stream 9 already.  You can't drop it from this file with the `c9s` label included.  You likely want to create a new eln specific workload and unwanted file instead.